static/src/javascripts/boot.js

```diff
@@ -1,101 +1,88 @@
-    require.ensure(
-        [],
-        require => {
-            // es7 polyfills not provided by pollyfill.io
-            require('core-js/modules/es7.object.values');
-            require('core-js/modules/es7.object.get-own-property-descriptors');
-            require('core-js/modules/es7.string.pad-start');
-            require('core-js/modules/es7.string.pad-end');
-
-            const domready = require('domready');
-            const raven = require('lib/raven');
-            const bootStandard = require('bootstraps/standard/main');
-            const config = require('lib/config');
-            const { markTime } = require('lib/user-timing');
-            const { capturePerfTimings } = require('lib/capture-perf-timings');
+    domready(() => {
+        // 1. boot standard, always
+        markTime('standard boot');
+        bootStandard();
 
-            domready(() => {
-                // 1. boot standard, always
-                markTime('standard boot');
-                bootStandard();
-
-                // 2. once standard is done, next is commercial
-                if (config.page.isDev) {
-                    window.guardian.adBlockers.onDetect.push(isInUse => {
-                        const needsMessage =
-                            isInUse && window.console && window.console.warn;
-                        const message =
-                            'Do you have an adblocker enabled? Commercial features might fail to run, or throw exceptions.';
-                        if (needsMessage) {
-                            window.console.warn(message);
-                        }
-                    });
+        // 2. once standard is done, next is commercial
+        if (config.page.isDev) {
+            window.guardian.adBlockers.onDetect.push(isInUse => {
+                const needsMessage =
+                    isInUse && window.console && window.console.warn;
+                const message =
+                    'Do you have an adblocker enabled? Commercial features might fail to run, or throw exceptions.';
+                if (needsMessage) {
+                    window.console.warn(message);
                 }
+            });
+        }
 
-                markTime('commercial request');
-                require.ensure(
-                    [],
-                    // webpack needs the require function to be called 'require'
-                    // eslint-disable-next-line no-shadow
-                    require => {
-                        raven.context(
-                            { tags: { feature: 'commercial' } },
-                            () => {
-                                markTime('commercial boot');
-                                const commercialBoot = config.switches
-                                    .commercial
-                                    ? require('bootstraps/commercial')
-                                    : Promise.resolve;
+        markTime('commercial request');
```

Comment:
i don't think webapck is clever enough to understand this. afaik, it handles the webpack-specific `require.ensure` calls by string matching